packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1062,9 +1062,15 @@ export const components = (
     MuiDialogActions: {
       styleOverrides: {
         root: ({ theme }) => ({
+          flexWrap: "wrap",
+          gap: odysseyTokens.Spacing2,
           paddingBlockStart: theme.spacing(4),
           paddingBlockEnd: theme.spacing(6),
           paddingInline: theme.spacing(6),
+
+          "& > *": {
```

Comment:
Does `& > .${buttonClasses.root}` work here instead?

Trying to figure out how to keep it specific and hopefully remove the need for `!important`.

Reply:
Oh yeah, that worked 🙌